hyperspy/gui/egerton_quantification.py

```diff
@@ -369,10 +369,12 @@ def _interpolator_kind_changed(self, old, new):
         self.span_selector_changed()
 
     def _ss_left_value_changed(self, old, new):
-        self.span_selector_changed()
+        if not np.isnan(self.ss_right_value):
+            self.span_selector_changed()
 
     def _ss_right_value_changed(self, old, new):
-        self.span_selector_changed()
+        if not np.isnan(self.ss_right_value):
```

Comment:
I think that this is correct: the idea is that, when just starting dragging the span selector, `ss_left_value` and `ss_right_value` will be updated with the same value (width=0) and we want to avoid triggering `span_selector_changed` in this situation. (This situation is what was causing the bug the this PR intends to fix). Do you disagree? If not, I'll add comments to explain this.